### ISSUE TITLE
Broken software

### ISSUE THREAD

**Issue by jjpe** [syncthing/syncthing#2215]
Since @AudriusButkevicius was so friendly to close the last issue without reading it properly, here it is again.

I see no way in either the manual or the software to actually make things sync. It looks like it but it's permanently out of sync. Please fix, as-is this whole thing is pointless.

I'm using the server on OS X and the Android app.
The workflow as far as I can figure it out:
1. add a folder to sync on the OS X web interface. Mark it as the folder master.
2. add a folder on the app with the same folder id
3. let 'er rip (well, crash and burn).

If this is not the intended workflow, the GUI needs a lot of work, since that is what it suggests to me.

**Comment by andywenk** [comment]
@jjpe just a side note: the issue was closed, because your tone was a bit offending. As I just read this issue without reading the original one, I thought the same. So maybe you try to be a bit more polite and ask friendly for help. Closing the issue with just saying "the issue is just a rant" is also not the gentleman's way on the other hand.

**Comment by jjpe** [comment]
Though I was somewhat irritated, the intention was merely to be direct, not to offend. 
I've tried to get it working for quite a while, following the instructions, and getting nowhere fast. 
My usual solution in such a situation is to submit a bug report that is short and sweet to get the bug sorted out quickly together so everyone can be on their merry way, the project itself included.
This time perhaps it was a bit too short though, so I added the rest of the inital post. By that time this was already a 'thing'.
